tools/helmify/src/config.yaml

```diff
@@ -44,14 +44,14 @@ katib:
       app_version: v0.14.0-rc.0
       
 
-istio-1-14:
+istio:
   kustomization_paths:
-    - upstream/common/istio-1-14/istio-crds/base
-    - upstream/common/istio-1-14/istio-namespace/base
-    - upstream/common/istio-1-14/istio-install/base
-  output_helm_chart_path: charts/common/istio-1-14
-  version: 0.1.1
-  app_version: 1.14.1
+    - upstream/common/istio-1-16/istio-crds/base
+    - upstream/common/istio-1-16/istio-namespace/base
+    - upstream/common/istio-1-16/istio-install/base
+  output_helm_chart_path: charts/common/istio-1-16
```

Comment:
didnt the folder get changed to istio? Ignore if the folder name change is temporary.

Reply:
good catch, addressed.